include/arch/arm/aarch32/cortex_m/mpu/arm_mpu_v7m.h

```diff
@@ -115,6 +115,11 @@
 	(NORMAL_OUTER_INNER_WRITE_BACK_WRITE_READ_ALLOCATE_NON_SHAREABLE | \
 	 MPU_RASR_XN_Msk | size | P_RW_U_NA_Msk) \
 }
+#define REGION_NOCACHE_ATTR(size) \
+{ \
+	(NORMAL_OUTER_INNER_NON_CACHEABLE_SHAREABLE | \
```

Comment:
@reloZid what is the reason this needs to be declared as Sharable?

Reply:
I also wonder, whether we could name it REGION_RAM_NOCACHE_ATTR, since we only seem to need this for RAM region configurations, that is, make the name more explicitly reflect the usage.

Reply:
@ionnisg Adopted this from an example from ST. But also seems to work as non-shareable. Applied both your suggestions.